test/useForm.test.tsx

```diff
@@ -166,17 +166,8 @@ describe('useForm', () => {
 
     act(() => {
       result.current.fields.name.onChange('Widget A')
-    })
```

Comment:
To give context on why the test update looks like this, `act(() => {})` is a React testing thing that says "hey do these side-effects within a render cycle". 

So previously, we were splitting our updates across render cycles. And now I collapsed everything into a single `act(() => {})` -- i.e. do all the updates in the same render cycle.